### ISSUE TITLE
Layout issues with keyboard predictions

### ISSUE THREAD

**Issue by kiliankoe** [KrauseFx/MajorKey#2]
OMG I've been needing this app for forever. I've been sending myself mails for years and have been playing with a similar idea every now and then, but definitely not quite as streamlined as this is. Thank you! ❤️ 

I am however running into a bit of a layout issue with the two buttons being obscured behind the keyboard text predictions on my iPhone 6s (not an X 😢).

![img_0207](https://user-images.githubusercontent.com/2625584/33277697-9323a106-d399-11e7-8ea8-74c4eff861ec.PNG)

**Comment by kiliankoe** [comment]
Oh, just checked. They're positioned absolutely 🙈 Should've had a look at that earlier^^

**Comment by KrauseFx** [comment]
Yeah, I only have an iPhone X, so I'm not gonna fix that (haha, I'm such a nice contributor), but that's why the app is open source, do you want to submit a PR to dynamically position the 🔑  button?

**Comment by kiliankoe** [comment]
I just removed both buttons in my locally deployed version instead 🙈😄 The keyboard's done button is perfectly fine^^

**Comment by KrauseFx** [comment]
Haha, yah that works also. Still, if you want to, I'd merge a PR 👍

**Comment by patricks** [comment]
Should be fixed with that PR 👉 #3
